### ISSUE TITLE
Codegen compatibility: Microsoft.CodeAnalysis.Scripting.Common 1.3.2

### ISSUE THREAD

**Issue by luomai** [dotnet/orleans#2554]
Hello,

I tried to reference the "Microsoft.CodeAnalysis.Scripting.Common 1.3.2" and "Microsoft.CodeAnalysis.CSharp.Scripting 1.3.2" into my Orleans project to speed up data processing within grains. However, these references crash my Silo startup and throw me an exception as follows:

> Exception thrown: 'System.IO.FileNotFoundException' in mscorlib.dll
> 
> Additional information: Could not load file or assembly 'System.Runtime.Loader, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.

My Orleans configuration is as follows:
Target framework: 4.6
Orleans 1.3.1
VS 2015

Is there anyway I can use these two packages within my Orleans project? 

Thanks in advanced,
Luo

**Comment by ReubenBond** [comment]
> to speed up data processing within grains
Do you mean that you want to perform your *own* kind of code generation within grains?

Is it possible that you could show your project layout & dependencies?

**Comment by DixonDs** [comment]
@luomai  Does it actually cause any issues other than an error on silo startup? I mean, does silo crash or continues startup anyway? It looks like related to https://github.com/dotnet/orleans/issues/1517, which was addressed by https://github.com/dotnet/orleans/pull/1534. The root cause is in Roslyn itself, and reported a while back: https://github.com/dotnet/roslyn/issues/9417.

We also use Orleans and Roslyn scripting, so I've just got used to see that error every time the silo is starting :)
